examples/infograph/infograph_trainer.py

```diff
@@ -1,6 +1,13 @@
+# !/usr/bin/env python
+# -*- encoding: utf-8 -*-
+"""
+@File    :   infograph_trainer.py
+@Time    :   2022/05/24 22:05:55
+@Author  :   Yang Yuxiang
+"""
 import os
 
-# os.environ['TL_BACKEND'] = 'paddle'
+os.environ['TL_BACKEND'] = 'paddle'
```

Comment:
I think there is no need to change this line. Please put this line into the comment.